packages/api/src/graphql/mutation.ts

```diff
@@ -320,12 +320,26 @@ export const GraphQLAdminMutation = new GraphQLObjectType<undefined, Context>({
       type: GraphQLUser,
       args: {
         id: {type: GraphQLNonNull(GraphQLID)},
-        password: {type: GraphQLNonNull(GraphQLString)}
+        password: {type: GraphQLNonNull(GraphQLString)},
+        sendMail: {type: GraphQLBoolean}
       },
-      resolve(root, {id, password}, {authenticate, dbAdapter}) {
+      async resolve(
+        root,
+        {id, password, sendMail},
+        {authenticate, sendMailFromProvider, dbAdapter}
+      ) {
         const {roles} = authenticate()
         authorise(CanResetUserPassword, roles)
-        return dbAdapter.user.resetUserPassword({id, password})
+        const user = await dbAdapter.user.resetUserPassword({id, password})
+        if (sendMail && user) {
+          await sendMailFromProvider({
+            recipient: user.email,
+            subject: 'Your password has been reset',
+            message: `Hello ${user.name}\n\nYour password has been reset. You can login with your new password.`,
```

Comment:
Should this have translation option?

Reply:
Yes it should. But like with the code above this is just for testing and will be replaced soon.